src/main/java/teammates/logic/core/LoggingLogic.java

```diff
@@ -0,0 +1,81 @@
+package teammates.logic.core;
+
+import java.time.Instant;
+import java.util.List;
+
+import com.google.cloud.MonitoredResource;
+import com.google.cloud.logging.LogEntry;
+import com.google.cloud.logging.Payload.StringPayload;
+import com.google.cloud.logging.Severity;
+
+import teammates.common.util.Config;
+import teammates.common.util.Logger;
+import teammates.logic.core.LoggingService.LogSearcher;
+
+/**
+ * Handles the logic related to logging.
+ */
+public final class LoggingLogic {
+
+    private static LoggingLogic instance = new LoggingLogic();
+
+    private static final Logger log = Logger.getLogger();
+
+    private static final String LOG_NAME = "audit-logs";
+    private static final Severity LOG_LEVEL = Severity.INFO;
+    private static final String LOG_TYPE_ACCESS = "access";
```

Comment:
If these are constants, maybe would be better to place them in `Const.java` or enum it